traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -444,11 +444,11 @@ AuthenticatedUrlRequest build(Token token) {
             String tokenValue = (null != token) ? token.getValue() : null;
             return new AuthenticatedUrlRequest(method, getURL(), headers, tokenValue) {
                 @Override
-                public JSONObject getJSONPayload() {
+                public String getPayload() {
```

Comment:
Does this still need to be a String? Can we keep this as a JSONObject?

Reply:
thsats what we talk about and decided to change to take json out from the router. now all router cares about is the string you want to send as post body. and it does not have to be json.

i was going to put a TODO. but it was easy